service/src/main/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneService.java

```diff
@@ -273,6 +276,41 @@ public List<UUID> listLandingZoneIds(BearerToken bearerToken, UUID billingProfil
         .toList();
   }
 
+  /**
+   * Lists landing zone record for a billing profile ID.
+   *
+   * @param bearerToken bearer token of the calling user.
+   * @param billingProfileId the billing profile ID to search.
+   * @return landing zone record.
+   */
+  public LandingZoneRecord getLandingZoneRecord(BearerToken bearerToken, UUID billingProfileId) {
+    // Call BPM to validate user access to billing profile.
+    bpmService.verifyUserAccess(bearerToken, billingProfileId);
```

Comment:
Not sure we need to verify access to the billing profile? `getLandingZoneRecordByBillingProfile` checks the user has access to the LZ itself -- that should be all we need to check, right?

Reply:
To clarify: in Sam the permissions inherit from the billing-profile to the landing-zone. So as a caller we can just check permissions against the LZ resource.

Reply:
Changed to use SAM checks for Landing Zone only.